docker/php/www.conf

```diff
@@ -440,4 +440,17 @@ pm.max_spare_servers = 3
 
 php_admin_flag[log_errors] = on
 
-php_admin_value[date.timezone] = Europe/Paris
\ No newline at end of file
+php_admin_value[date.timezone] = Europe/Paris
+
+;Debug Settings
+php_admin_value[xdebug.mode] = develop
+php_admin_value[debug.start_with_request] = yes
+php_admin_value[xdebug.log]="/tmp/xdebug.log"
+php_admin_value[debug.show_local_vars] = 1
```

Comment:
Shouldn't it be xdebug.show_local_vars ? ([Official documentation](https://xdebug.org/docs/all_settings#show_local_vars))

Maybe this is why we don't see full local variables.